docs/pages/quick-start.mdx

```diff
@@ -24,13 +24,9 @@ You'll be prompted to pick the type of template to use (vanilla JS or React).
 
 ### Run your project
 
-Jump into your new project's directory. In one terminal, run the anvil node with:
+Jump into your new project's directory.
 
-```
-pnpm run dev:node
-```
-
-And in another terminal, run the app with:
+Run the development server with:
```

Comment:
Can't do a suggestion over these lines because of the deletion, but this can probably be one line

> Jump into your new project's directory and run the app with: